src/data/faq.json

```
@@ -352,10 +352,10 @@
                         ]
                     },{
                         "active": true,
-                        "title": "[Google/Android]: Notification of possible encounters... - Weekly Update",
+                        "title": "'COVID-19 exposure logging... - Weekly Update'",
                         "anchor": "notif_weekly_update",
                         "textblock": [
-                            "This is a general notification. It does not mean that exposures were detected. The Exposure Notification Framework sends this notification. It looks like other notifications of the Corona-Warn-App, for example, when Bluetooth is deactivated. You can look up the risk status directly in the Corona-Warn-App."
+                            "This is a general notification. It does not mean that exposures were detected. The Exposure Notification Framework sends this notification and displays the icon. You can look up the current risk status directly in the Corona-Warn-App."
```

which translation/text is "leading"/"primary/"canonical"?

The german text is
`Sie können Ihren Risikostatus direkt in der Corona-Warn-App einsehen.`.

The english text is
`You can look up the current risk status directly in the Corona-Warn-App.`

Note `Ihren Risikostatus` - `the current risk status`.

Assuming german is leading, `Ihren` would translate to `your`:
```suggestion
                            "This is a general notification. It does not mean that exposures were detected. The Exposure Notification Framework sends this notification and displays the icon. You can look up your current risk status directly in the Corona-Warn-App."
```